plugins/interactivity-api-countdown-3cd73e/interactivity-api-countdown-3cd73e.php

```diff
@@ -4,6 +4,7 @@
  * Description:       Countdown created with the Interactivity API
  * Requires at least: 6.1
```

Comment:
Heard, we could support backwards compatibility, pre-6.5 with a conditional
> IF < WordPress 6.5
> Requires Plugin: gutenberg

otherwise act normal?

Realize this is a broader conversation about supporting pre 6.5 sites..

Reply:
@ryanwelcher - Can you please assign this item to me?

> In this case, I think that we should use the Requires at least header and set it to 6.5

Good idea, why create support for something that is brand new for backwards compatibility. Doesn't make much sense :)

If you can assign this to me, I'll add it to the backlog